pep-0517.txt

```diff
@@ -166,48 +166,71 @@ The build backend object is expected to have attributes which provide
 some or all of the following hooks. The common ``config_settings``
 argument is described after the individual hooks::
 
-  def get_build_requires(config_settings):
+  def build_sdist(source_directory, sdist_directory, config_settings):
+      ...
+
+MUST build an unpacked sdist from the ``source_directory`` and place it into the
+specified ``sdist_directory``. If the ``source_directory`` is already an
```

Comment:
We have a ```build_wheel``` hook and a ```get_wheel_metadata``` hook. But here, we just have ```build_sdist``` - is the backend expected to create the sdist metadata in ```sdist_directory```? Why the discrepancy? And doesn't this require backends to track future "sdist >=2.0" specs to maintain correct metadata? I think we should decouple creating sdist metadata from backends. And once we get into this discussion, we're opening up the whole "sdist metadata" debate, which is nowhere near resolution. Hence my preference for a "just give me the files required to build" hook.